CRM/Core/Config.php

```diff
@@ -287,7 +287,9 @@ public function cleanupCaches($sessionReset = TRUE) {
     UserJob::delete(FALSE)->addWhere('expires_date', '<', 'now')->execute();
     // clear all caches
     self::clearDBCache();
-    Civi::cache('session')->clear();
+    // [ML] SYMBIOTIC This causes problems if people are filling-in quickform
+    // forms during a system.flush
+    // Civi::cache('session')->clear();
```

Comment:
In this context, it receives the param `$sessionReset`. Notwithstanding the method-signature here, the actual/typical default should `FALSE`. (For purposes of `cv flush`, `System.flush`, `civicrm/menu/rebuild`, etc -- they all call via `rebuildMenuAndCaches(...$sessionReset=FALSE)`). It seems better to abide that flag, no?

Reply:
if we change the param `$sessionReset = FALSE` by default, then add an "if" around ` Civi::cache('session')->clear()`, then yes, it works as intended I think.

Tested with:

- `cv flush`
- "flush cache" from the admin UI
- `cv api system.flush session=1` => will flush the QF cache (sessionReset=true)